cloud.dev.deploy.yaml

```diff
@@ -1,14 +1,44 @@
 steps:
-- name: 'gcr.io/cloud-builders/npm'
-  args: ['install']
-- name: 'gcr.io/cloud-builders/npm'
-  args: ['run', 'test']
-- name: 'gcr.io/cloud-builders/npm'
-  args: ['run', 'build']
-- name: 'gcr.io/cloud-builders/docker'
-  args: ['build', '-t', 'gcr.io/$PROJECT_ID/effektapi-dev:latest', '.']
-- name: 'gcr.io/cloud-builders/docker'
-  args: ['push', 'gcr.io/$PROJECT_ID/effektapi-dev:latest']
-- name: 'gcr.io/cloud-builders/gcloud'
-  args: ['run', 'deploy', 'effektapi-dev', '--image', 'gcr.io/$PROJECT_ID/effektapi-dev:latest', '--region', 'europe-north1', '--platform', 'managed', '--allow-unauthenticated']
+  - name: 'gcr.io/cloud-builders/npm'
+    args: ['install']
+  - name: 'gcr.io/cloud-builders/npm'
+    args: ['run', 'test']
+  - name: 'gcr.io/cloud-builders/npm'
+    args: ['run', 'build']
+  - name: 'gcr.io/cloud-builders/docker'
+    args: ['build', '-t', 'gcr.io/$PROJECT_ID/effektapi-dev:latest', '.']
+  - name: 'gcr.io/cloud-builders/docker'
+    args: ['push', 'gcr.io/$PROJECT_ID/effektapi-dev:latest']
+    id: 'build'
+
+  - name: 'gcr.io/cloud-builders/gcloud'
+    id: 'deploy-norway'
+    waitFor: ['build']
+    args:
+      - 'run'
+      - 'deploy'
+      - 'effektapi-dev'
+      - '--image'
+      - 'gcr.io/$PROJECT_ID/effektapi-dev:latest'
+      - '--region'
+      - 'europe-north1'
+      - '--platform'
+      - 'managed'
+      - '--allow-unauthenticated'
+
+  - name: 'gcr.io/cloud-builders/gcloud'
+    id: 'deploy-sweden'
+    waitFor: ['build']
+    args:
+      - 'run'
+      - 'deploy'
+      - 'geeffektivt-dev'
+      - '--image'
+      - 'gcr.io/$PROJECT_ID/effektapi-dev:latest'
+      - '--region'
+      - 'europe-north1'
+      - '--platform'
+      - 'managed'
+      - '--allow-unauthenticated'
+
```

Comment:
Example run: https://console.cloud.google.com/cloud-build/builds;region=global/7cf8fce0-c6ea-4ecb-8b82-f9e7d85ff372;tab=detail?authuser=1&project=hidden-display-243419

(NO failing here for some reason though)

Reply:
I don't think it affects stability, but allows the deploys to happen in parallel

Reply:
The failing :norway: deploy has been fixed. Turned out to be unrelated